docs/examples/MarrmotM01.ipynb

```diff
@@ -2,196 +2,142 @@
  "cells": [
```

Comment:
<p>Can we add a <code>## Running the model</code> heading above this cell? As we are no longer setting up the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Comments and headers are revised. </p>